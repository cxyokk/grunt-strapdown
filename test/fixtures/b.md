# i am test b
## i am sleeping early

blahblahblah...
