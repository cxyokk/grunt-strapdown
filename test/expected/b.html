<!DOCTYPE html>
<html>
<title>b</title>
<body>
<xmp theme="cyborg" style="display:none;">
# i am test b
## i am sleeping early

blahblahblah...

</xmp>
<script src="http://strapdownjs.com/v/0.2/strapdown.js"></script>
</body>
</html>
